src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Driver olga = new Driver();
+        olga.setName("Olga");
+        olga.setLicenseNumber("5678");
+        olga.setId(10L);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        driverService.create(olga);
+        System.out.println("Get all drivers from drivers table"
+                + driverService.getAll());
+        System.out.println("Get driver with id = 5 from drivers table "
```

Comment:
why id = 5? if you get id = 1?

Reply:
because in my table after tests I have a lot of drivers))